src/main.ts

```diff
@@ -4,6 +4,9 @@ import 'core-js/stable';
 // NOTE: this is not working for styleguidist. There you need to add style imports to the required configuration
 import './setup/_scss.scss';
 
+// vendor styles
+import 'swiper/css';
```

Comment:
Can't we import them inside of `./setup/_scss.scss`?

Reply:
No, the import does not seem to work in scss: 
![image](https://user-images.githubusercontent.com/3381820/144189615-59cee498-d7a0-47ff-968c-c86b0d993dbc.png)
The changed the structure and Package Type of swiper, see here: https://swiperjs.com/migration-guide